tuning/data/data_handlers.py

```diff
@@ -328,57 +366,72 @@ def tokenize(
 
 def duplicate_columns(
```

Comment:
Instead of thinking of this as only duplicate, let's look at normal data frame operations that's users would want and support them.

Reply:
Could you explain what others you had in mind?

Reply:
I guess you already have many of them as native functions, so we should be good.

"remove_columns": DataHandler(
        # Native function
        handler_type=DataHandlerType.REMOVE,
    ),
    "select_columns": DataHandler(
        # Native function
        handler_type=DataHandlerType.SELECT,
    ),
    "rename_columns": DataHandler(
        # Native function
        handler_type=DataHandlerType.RENAME,
    ),